src/index.html

```diff
@@ -1,24 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Output Management</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-  <body>
-    <div class="container">
-      <div class="list">
-        <h2>To-do list</h2>
-        <div class="row">
-          <input type="text" id="input-box" placeholder="add to your list" />
-          <button class="add-btn">Add</button>
-        </div>
-        <ul id="list-container"></ul>
-        <button class="removeButton">clear</button>
-      </div>
-    </div>
-    <script type="module" src="index.js"></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Output Management</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="list">
+        <h2>To-do list</h2>
+        <div class="row">
+          <input type="text" id="input-box" placeholder="add to your list" />
+          <button class="add-btn">Add</button>
+        </div>
+        <ul id="list-container"></ul>

```

Comment:
- I noticed you stored tasks as an `html list element (li)` in the localStorage, This should not be as it will not enable you to perform the project requirement that states: `it should contain methods related to the status updates (`completed`: `true` / `false`).`  , I will advise you to use and array of objects to store each task, stringify them and store them in the local storage, the object might look like the code snippet below for each task:
```javascript
[
{
id: 1, 
task: "The task goes here",
isCompleted: false, 
}
]
``` 

- Please go through the project requirements carefully to understand and implement them appropriately. 


- **Image of how you stored tasks in local storage***
![image](https://github.com/LethaboMot/weppack2/assets/58771507/20eb4f9d-f2cb-478c-9f97-a87483b48dad)